palvelutarjotin/settings.py

```diff
@@ -175,6 +175,7 @@
     "utils",
     "organisations",
     "occurrences",
+    "verification_token",
```

Comment:
I first started implementing tokens to occurrences, but then I felt that it's almost the same to create them more generic. Now different kind of verification tokens can be created for any kind of model instances.